src/client/components/Forms/ResetPassword.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+export default function ResetPassword({ onSubmit }) {
+  const [email, setEmail] = useState('');
+  const handleEmailInput = (e) => setEmail(e.target.value);
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    await onSubmit({ email });
+  };
+  return (
+    <form onSubmit={handleSubmit}>
```

Comment:
Just curious, but why not just call onSubmit here with the email, and let the onSubmit function do the preventDefault?

Reply:
I just prefer to extract the functions out of the "rendering" part :) if that answers your question? :)